valgrind/launch-test-instance

```diff
@@ -13,10 +13,10 @@ echo "Starting an instance..."
 valgrind_instance_id=$(aws ec2 run-instances \
     --image-id ami-f4cc1de2 \
     --count 1 \
-    --instance-type m3.xlarge \
+    --instance-type m5.4xlarge \
```

Comment:
This instance type cannot be initialized without VPC set-up in the account. (https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-vpc.html#vpc-only-instance-types) 
This makes the scripts not runnable for the time being. I get the following error message
```
An error occurred (VPCResourceNotSpecified) when calling the RunInstances operation: The specified instance type can only be used in a VPC. A subnet ID or network interface ID is required to carry out the request.
```

We can go with c3.8xlarge or r3.2xlarge. Both has similar RAM.

Reply:
After changing it to r3.2xlarge, tests started successfully. Waiting for the results.

Reply:
We can create a VPC and destroy it after we are done with the tests, but I guess r3 instances would also work for our purpose. So I'm inclined to just change instance type to r3.2xlarge after your tests are passed.